### PR TITLE
8172 update the right drawer action menu to open with command o

### LINKED ISSUE #8172: Update the right drawer action menu to open with command O
## Scope & Context

Capture d’écran 2024-10-29 à 14 40 49

https://www.figma.com/design/PNBfTgOVraw557OXChYagk/Explo?node-id=23016-114699&t=birJ7NQBEHqdnuTr-4

## Technical inputs

Modify the right drawer action menu and introduce a button which opens a dropdown when clicked or when the user uses `⌘O`.

### PR DESCRIPTION
**Pull request by bosiraphael**
Closes #8172 

- Added a shortcut property to the button component
- Displays the actions inside a dropdown
- The dropdown is toggled either by clicking on the button or with the `command + O` shortcut

https://github.com/user-attachments/assets/4c4c88fa-85dc-404e-bb42-f2b0d57c8960

### CODE REVIEW

File: packages/twenty-front/src/modules/action-menu/components/RecordShowRightDrawerActionMenu.tsx [hunk 1]
```diff
@@ -13,20 +13,20 @@ export const RecordShowRightDrawerActionMenu = () => {
   );
 
   return (
-    <>
+    <ActionMenuContext.Provider
```
Comment (charlesBochet) [inline packages/twenty-front/src/modules/action-menu/components/RecordShowRightDrawerActionMenu.tsx:16]: why this change?

File: packages/twenty-ui/src/input/button/components/Button.tsx [hunk 2]
```diff
@@ -358,6 +359,18 @@ const StyledSoonPill = styled(Pill)`
   margin-left: auto;
 `;
 
+const StyledShortcutLabel = styled.div`
+  color: ${({ theme }) => theme.font.color.light};
+  font-weight: ${({ theme }) => theme.font.weight.medium};
+`;
+
+const StyledSeparator = styled.div<{ size: 'sm' | 'md' }>`
```
Comment (charlesBochet) [inline packages/twenty-ui/src/input/button/components/Button.tsx:367]: what's this size?

File: packages/twenty-ui/src/input/button/components/__stories__/Button.stories.tsx [hunk 3]
```diff
@@ -197,6 +200,84 @@ export const PositionCatalog: CatalogStory<Story, typeof Button> = {
     fullWidth: { control: false },
     soon: { control: false },
     position: { control: false },
+    shortcut: { control: false },
+  },
+  parameters: {
```
Comment (charlesBochet) [inline packages/twenty-ui/src/input/button/components/__stories__/Button.stories.tsx:205]: maybe we don't need to have two full catalog
the shortcut one could only have 1 dimension which is the shortcut

**Review by charlesBochet — APPROVED** [review]

File: packages/twenty-front/src/modules/action-menu/components/RecordShowRightDrawerActionMenu.tsx (hunk 1 above)
Comment (bosiraphael) [reply packages/twenty-front/src/modules/action-menu/components/RecordShowRightDrawerActionMenu.tsx:16]: it's not really necessary

File: packages/twenty-ui/src/input/button/components/Button.tsx (hunk 2 above)
Comment (bosiraphael) [reply packages/twenty-ui/src/input/button/components/Button.tsx:367]: The height of the separator
